set.mm

```diff
@@ -131380,6 +131380,18 @@ used to convert hypothesis of the inference (deduction) form of this
       ( comraddd mvrraddd ) ABDCFEABCDEFGHI $.
   $}
 
+  ${
+    laddrotrd.a $e |- ( ph -> A e. CC ) $.
```

Comment:
Is it necessary to add such a trivial variant of mvlladdd and mvrladdd to main?  Using a very different label?  Can we expect this theorem will make at least 7 theorems shorter, if applied consequently? Norm Megill once mentioned the number 7 as his personal limit that must be surpassed before he added a trivial modification (convenience) to the database.

Reply:
Indeed it's quite trivial, I will avoid using it, and use `mvlladdd` directly instead.

Very often the next step I use is some variation on `eqtrd`, which can easily handle commutativity of equalities simply by switching to a different variation, like in `eqtrrd`.